README.md

```diff
@@ -9,104 +9,13 @@ Triton is a simple DNS client made for better understanding of DNS protocol
 ## From this repo
 
 ```bash
-git clone https://git.best-service.online/yurzs/triton.git  
+git clone https://github.com/Yurzs/triton
 cd triton 
 python3 setup.py install
 ```
 
 ## How to use
 
-Here is an example query for A record of this gitlab instance
```

Comment:
If you remove old example please provide another one for replacement

Reply:
which function should I use for the example?  Most usages are not simple for using on the REPL

Reply:
ping @Yurzs

Reply:
Use the same methods as in deleted section.
You won't be able to use it from web repl services because they forbid access to 53 port. Run it on local machine.